client/graphql.schema.json

```diff
@@ -953,6 +945,33 @@
         "enumValues": null,
         "possibleTypes": null
       },
+      {
+        "kind": "OBJECT",
+        "name": "ChapterUsersCount",
+        "description": null,
+        "fields": [
+          {
+            "name": "chapter_users",
+            "description": null,
+            "args": [],
+            "type": {
+              "kind": "NON_NULL",
+              "name": null,
+              "ofType": {
+                "kind": "SCALAR",
+                "name": "Float",
+                "ofType": null
```

Comment:
Noob question, do you know why this type of data is float, and not int?

and will we have to deal with a user in the other side of the decimal? and how? 🤕

Reply:
That's because I used `Number` instead of `Int` in the graphql type.